packages/position/src/react/index.js

```diff
@@ -88,7 +91,7 @@ const Position = React.forwardRef((props, forwardedRef) => {
 
   return (
     <>
-      {target ? child : React.cloneElement(child, { ref })}
```

Comment:
Apparently this didn't work to pass ref as a prop.  Did you learn why?  If so, would you mind explaining for my benefit, please?

Reply:
this overwrites the ref with the new ref we're providing.

Reply:
ah